MapboxCoreNavigation/SpokenInstructionFormatter.swift

```diff
@@ -72,9 +72,33 @@ public class SpokenInstructionFormatter: NSObject {
         // We only want to announce this special depature announcement once.
         // Once it has been announced, all subsequnt announcements will not have an alert level of low
         // since the user will be approaching the maneuver location.
-        let isDeparture = routeProgress.currentLegProgress.currentStep.maneuverType == .depart && alertLevel == .depart
-        let didCompleteManeuver = routeProgress.currentLegProgress.currentStep.distance > 2_000 && routeProgress.currentLegProgress.alertUserLevel == .low
-        if isDeparture || didCompleteManeuver {
+        let isDeparture = routeProgress.currentLegProgress.currentStep.maneuverType == .depart && (alertLevel == .depart || alertLevel == .low)
+        if let currentInstruction = currentInstruction, isDeparture {
+            if routeProgress.currentLegProgress.currentStep.distance > 2_000 {
```

Comment:
Pull this number out into a constant and document it.

Reply:
Should this be based on the step's overall distance or the distance remaining to the maneuver? Also, what happens if the user is a significant distance from the departure point, like in a parking lot – how does that affect the behavior?

Reply:
> Also, what happens if the user is a significant distance from the departure point, like in a parking lot – how does that affect the behavior?

Let's pull this out into a different PR.